util/argconfig.c

```diff
@@ -389,11 +402,15 @@ int argconfig_parse(int argc, char *argv[], const char *program_desc,
 	long_opts[option_index++].val = 'h';
 
 	long_opts[option_index].name = "json";
-	long_opts[option_index].val = 'j';
+	long_opts[option_index++].val = 'j';
```

Comment:
This looks odd. The output format is handled via the `-o` `--output` options, which not all command currently support. So having this here doens't look right. We should remove it.